# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../hostap/hostap-driver_nodepmod.diff
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
diff -ruN hostap-driver-0.2.1/Makefile hostap-driver-0.2.1-new/Makefile
--- hostap-driver-0.2.1/Makefile	2004-04-29 06:39:27.000000000 +0200
+++ hostap-driver-0.2.1-new/Makefile	2004-06-18 16:02:07.449699816 +0200
@@ -47,7 +47,7 @@
 
 VERFILE := $(KERNEL_PATH)/include/linux/version.h
 KERNELRELEASE := $(shell if [ -r $(VERFILE) ]; \
-	then (cat $(VERFILE); echo UTS_RELEASE) | $(CC) $(INCLUDES) $(CFLAGS) -E - | tail -1 | xargs echo; \
+	then (cat $(VERFILE); echo UTS_RELEASE) | $(CC) $(INCLUDES) $(CFLAGS) -E - | tail -n 1 | xargs echo; \
 	else uname -r; fi)
 KERNELVER := $(shell echo "$(KERNELRELEASE)" | \
 	sed "s/\([0-9]*\.[0-9]*\.[0-9]*\).*/\1/")
@@ -62,6 +62,7 @@
 
 MSRC=driver/modules
 
+
 # if Rules.make exists in the kernel tree, we assume 2.4 style modules
 # if it doesn't assume 2.6 style
 OLDMAKE = $(wildcard $(KERNEL_PATH)/Rules.make)
@@ -90,6 +91,7 @@
 endif
 
 2.4:
+	echo "Kernelver: $(KERNELVER) Kernelrelease: $(KERNELRELEASE) Modpath: $(MODPATH)"
 	@if ! echo "$(KERNELRELEASE)" | grep -q "^2\.4"; then \
 		echo "*** Can't build for 2.4 with a non-2.4 source!"; \
 		exit 1; \
@@ -98,6 +100,7 @@
 	$(MAKE) install_all_info
 
 2.6:
+	echo "Kernelver: $(KERNELVER) Kernelrelease: $(KERNELRELEASE) Modpath: $(MODPATH)"
 	@if ! echo "$(KERNELRELEASE)" | grep -q "^2\.6" && \
 	    ! echo "$(KERNELRELEASE)" | grep -q "^2\.5"; then \
 		echo "*** Can't build for 2.6 with a non-2.6 source!"; \
@@ -142,12 +145,7 @@
 	@if [ -r $(MODPATH)/kernel/drivers/net/wireless/hostap_crypt.ko ]; then \
 		echo "You may need to remove old $(MODPATH)/kernel/drivers/net/wireless/hostap_crypt.ko"; \
 	fi
-	@if [ -f "/boot/System.map-$(KERNELRELEASE)" ] || [ "$(CURRENTKV)" = "$(KERNELRELEASE)" ]; then \
-		echo "*** Running depmod..."; \
-		/sbin/depmod -ae; \
-	else \
-		echo "*** I won't run depmod because I can't find an appropriate System.map file or you are not running the same kernel as the one you are building against. Please run it manually when the conditions are correct..."; \
-	fi
+	echo "ROCKLinux: not running depmod in build."
 
 install_hostap: $(MSRC)/hostap.o
 	@echo "Installing hostap.o to $(MODPATH_PCI)"
@@ -167,7 +165,7 @@
 	mkdir -p $(MODPATH_CS)
 	cp -f $(MSRC)/$(MODULE_CS) $(MODPATH_CS)
 ifndef DESTDIR
-	/sbin/depmod -ae
+	echo "ROCKLinux: not running depmod in build."
 endif
 
 install_conf:
@@ -183,13 +181,13 @@
 	@echo "Installing $(MODULE_PLX) to $(MODPATH_PLX)"
 	mkdir -p $(MODPATH_PLX)
 	cp -f $(MSRC)/$(MODULE_PLX) $(MODPATH_PLX)
-	/sbin/depmod -ae
+	echo "ROCKLinux: not running depmod in build."
 
 install_pci: install_check install_crypt install_hostap
 	@echo "Installing $(MODULE_PCI) to $(MODPATH_PCI)"
 	mkdir -p $(MODPATH_PCI)
 	cp -f $(MSRC)/$(MODULE_PCI) $(MODPATH_PCI)
-	/sbin/depmod -ae
+	echo "ROCKLinux: not running depmod in build."
 
 uninstall:
 	find $(MODPATH) -name "hostap*" | xargs rm -f
